utils/downloads.py

```diff
@@ -31,7 +32,7 @@
 from sounds.models import License
 
 
-def download_sounds(sounds_list, sounds_url=None):
+def download_sounds(sounds_list, sounds_url=None, licenses_url=None):
```

Comment:
Is this method used anywhere other than the `packs_download` method? We shouldn't have optional arguments if we don't need this method somewhere called with these arguments as `None`